kernel/api/src/main/java/org/sakaiproject/util/api/ValueEncryptionService.java

```diff
@@ -0,0 +1,21 @@
+package org.sakaiproject.util.api;
+
+public interface ValueEncryptionService {
```

Comment:
I think it's name is actually good (ValueEncryptionService) as it's clear what's it's doing, I just don't see the value in making it available for the whole of Sakai at the moment.

The static properties should remain de-coupled from the ValueEncryptionService, can they just be constants on the provider implementation?